source/unified-test-format/tests/Makefile

```diff
@@ -32,6 +32,9 @@ crud: HAS_AJV
 collection-management: HAS_AJV
 	@ajv test -s $(SCHEMA) -d "../../collection-management/tests/*.yml" --valid
 
+sessions: HAS_AJV
```

Comment:
FYI: "sessions" should also have been added to the `all` target above. See: https://github.com/mongodb/specifications/pull/1026#discussion_r660939153